server/src/integration-test/java/com/scalar/db/server/ConsensusCommitAdminIntegrationTestWithDistributedTransactionAdminService.java

```diff
@@ -41,6 +44,8 @@ protected void initialize(String testName) throws IOException {
       properties.setProperty(ServerConfig.PORT, PORT_FOR_SERVER_WITH_INCLUDE_METADATA_ENABLED);
       serverWithIncludeMetadataEnabled = new ScalarDbServer(properties);
       serverWithIncludeMetadataEnabled.start();
+    } else {
+      isExternalServerUsed = true;
```

Comment:
I'm not really sure what the external server means here. Is it true when the Scalar DB server is used in the tests?

Reply:
By default, we start the Scalar DB Server in the integration test JVM internally and run the tests on it. But when we specify the JVM option `scalardb.server.external_server_used=true`, we don't start the Scalar DB Server in the integration test JVM and run the tests on the external server.